tests/sp2n/Test_sp2n_lie_gen.cc

```diff
@@ -4,7 +4,7 @@ using namespace Grid;
 
 int main(int argc, char** argv) {
   Grid_init(&argc, &argv);
-
+    
```

Comment:
The list is starting to get long here. I know that iterating over template arguments is not exactly trivial in C++ but should we give it a try? [This](https://stackoverflow.com/questions/66664955/looping-over-a-templated-c-function-with-int-type/66665259#66665259) should work, although instead of templated lambdas we'd need another function template.

I assume that the `printGenerators` is missing later on because the matrices get too big?

Reply:
I guess the simplest here would be to wrap all the printing and calling of `testGenerators` into a function and then call that function a few times. Again... not quite a for loop but significantly less error-prone.